Makefile

```diff
@@ -158,3 +158,32 @@ restore-data-from-nightly-backup: read-deployment-config read-keyvault-config #
 	$(eval BACKUP_ARCHIVE_FILENAME=$(shell bin/download-nightly-backup ${backup_storage_secret_name} ${key_vault_name} ${paas_env}-db-backup ${paas_env}_backup- ${BACKUP_DATE}))
 	$(if $(CONFIRM_RESTORE), , $(error Restore can only run with CONFIRM_RESTORE))
 	bin/restore-nightly-backup ${space} ${postgres_database_name} ${BACKUP_ARCHIVE_FILENAME}
+
+get-image-tag:
+	$(eval export TAG=$(shell cf target -s ${space} 1> /dev/null && cf app teacher-training-api-${paas_env} | grep -Po "docker image:\s+\S+:\K\w+"))
+	@echo ${TAG}
+
+get-postgres-instance-guid: ## Gets the postgres service instance's guid make qa get-postgres-instance-guid
+	$(eval export DB_INSTANCE_GUID=$(shell cf target -s ${space} 1> /dev/null && cf service teacher-training-api-postgres-${paas_env} --guid))
+	@echo ${DB_INSTANCE_GUID}
+
+rename-postgres-service: ## make qa rename-postgres-service
+	cf target -s ${space} 1> /dev/null
+	cf rename-service teacher-training-api-postgres-${paas_env} teacher-training-api-postgres-${paas_env}-old
+
+remove-postgres-tf-state: set-tfenv deploy-init ## make qa remove-postgres-tf-state PASSCODE=xxxx
```

Comment:
This adds a dependency on tfenv. It's easier for us but since the goal is to let any dev do it, we should minimise dependencies.

Reply:
Removed set-tfenv and updated docs